### DISCUSSION TITLE
Tell me your favorite Sports!!

### DISCUSSION THREAD

**Discussion by nf-1** [NAIST-SE/AttractingContributors#1031] [General]
https://github.com/nf-1/AttractingContributors-sports

Please share with me some interesting sports!!

**Comment by Samabena** [comment]
Hello @nf-1  , I recently made a contribution to your project , could you please contribute to my as well ? Following is the link to [my repository](https://github.com/Samabena/sighting) . Thanks in advance.

**Reply by nf-1** [reply]
Of course! I did it.

**Comment by mustafizur-r** [comment]
Hello [@nf-1 ](https://github.com/nf-1) , I recently made a contribution to your project , I have added “Cricket" could you please contribute to my as well ? Following is the link to [my repository](https://github.com/MustafizurRU/AttractingContributors-My_Favorite_Sci-Fi_Books) . Thanks in advance.

**Reply by nf-1** [reply]
Of course! I did it.

**Comment by polarbearpppp** [comment]
Hi, I'm contribute to your project , Would you mind to contribute my project
https://github.com/polarbearpppp/AttractingContributors-Games
